docs/_docs/ruby-101.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Ruby 101
+permalink: /docs/ruby-101/
+---
+
+Jekyll is written in Ruby so you need to understand some of the jargon to reach its full potential.
```

Comment:
I really like that this page is here to educate people on the underling language that powers Jekyll and in turn their static site. But I'm worried it might put off people that are scared or really unfamiliar with Ruby? Maybe this item should be lower down the Getting Started nav section and the wording here should not mention a "need" if that makes sense? More like 
> "Having an understanding of some of the Ruby 'jargon' used in Jekyll could allow you to reach its full potential"

Reply:
There comes a time you got to face your fears, and you better face them early.

Reply:
I can see it both ways. I wanted new Jekyllers to be aware of these terms because they'll come up in installation, but then they probably won't use a `Gemfile` until they start using plugins.